pom.xml

```diff
@@ -2918,9 +2918,6 @@
                 <param>com.puppycrawl.tools.checkstyle.checks.NoCodeInFileCheck*</param>
                 <param>com.puppycrawl.tools.checkstyle.checks.OrderedPropertiesCheck*</param>
                 <param>com.puppycrawl.tools.checkstyle.checks.OuterTypeFilenameCheck*</param>
-                <param>
-                  com.puppycrawl.tools.checkstyle.checks.sizes.RecordComponentNumberCheck*
```

Comment:
https://github.com/checkstyle/checkstyle/actions/runs/3851529734/jobs/6562806022#step:6:337

This is the only one I am seeing not directly in `com.puppycrawl.tools.checkstyle.checks`. I did not recheck the others listed.